### PR TITLE
[tradfri] Added basic support for IKEA blinds FYRTUR and KADRILJ

### PR DESCRIPTION
**Pull request by cweitkamp**
- Added basic support for IKEA blinds FYRTUR and KADRILJ

Supersedes #6167 

Test version: [org.openhab.binding.tradfri-2.5.2-SNAPSHOT.zip](https://github.com/openhab/openhab-addons/files/4174200/org.openhab.binding.tradfri-2.5.2-SNAPSHOT.zip)

Also-by: Manuel Raffel <manidu@outlook.com>
Signed-off-by: Christoph Weitkamp <github@christophweitkamp.de>

### CODE REVIEW

File: bundles/org.openhab.binding.tradfri/src/main/java/org/openhab/binding/tradfri/internal/discovery/TradfriDiscoveryService.java [hunk 1]
```diff
@@ -43,26 +48,28 @@
...
-    private final TradfriGatewayHandler handler;
+    private @Nullable TradfriGatewayHandler handler;
 
     private static final String REMOTE_CONTROLLER_MODEL = "TRADFRI remote control";
 
-    private static final String[] COLOR_TEMP_MODELS = new String[] { "TRADFRI bulb E27 WS opal 980lm",
-            "TRADFRI bulb E27 WS clear 950lm", "TRADFRI bulb GU10 WS 400lm", "TRADFRI bulb E14 WS opal 400lm",
-            "FLOALT panel WS 30x30", "FLOALT panel WS 60x60", "FLOALT panel WS 30x90",
-            "TRADFRI bulb E12 WS opal 400lm" };
+    private static final List<String> COLOR_TEMP_MODELS = Collections
```
Comment (J-N-K) [inline bundles/org.openhab.binding.tradfri/src/main/java/org/openhab/binding/tradfri/internal/discovery/TradfriDiscoveryService.java]: Why not use a `Set` here?

Comment (cweitkamp) [reply bundles/org.openhab.binding.tradfri/src/main/java/org/openhab/binding/tradfri/internal/discovery/TradfriDiscoveryService.java]: Yes, good point. Why not use a `Set` here. I changed it.

**Comment by J-N-K** [comment]
Travis fails in itest (doesn't run at all). How do we want to handle that? I have seen that in a lot (all?) of PR lately when it comes to itests.

**Comment by cweitkamp** [comment]
At least for TRADFRI we ignored them last time (e.g. https://github.com/openhab/openhab-addons/pull/6544).

**Pull request merged**